v-next/hardhat-keystore/src/internal/hook-handlers/config.ts

```diff
@@ -0,0 +1,43 @@
+import type { ConfigHooks } from "@ignored/hardhat-vnext/types/hooks";
+
+import { validateUserConfigZodType } from "@ignored/hardhat-vnext-zod-utils";
+import { z } from "zod";
+
+import { getKeystoreFilePath } from "../utils/get-keystore-file-path.js";
+
+const keystoreConfigType = z.object({
+  filePath: z.string().optional(),
+});
+
+const userConfigType = z.object({
+  keystore: z.optional(keystoreConfigType),
+});
+
+export default async (): Promise<Partial<ConfigHooks>> => {
+  const handlers: Partial<ConfigHooks> = {
+    validateUserConfig: async (userConfig) => {
+      return validateUserConfigZodType(userConfig, userConfigType);
+    },
+    resolveUserConfig: async (
+      userConfig,
+      resolveConfigurationVariable,
+      next,
+    ) => {
+      const resolvedConfig = await next(
+        userConfig,
+        resolveConfigurationVariable,
+      );
+
+      const defaultKeystoreFilePath = await getKeystoreFilePath();
+
+      return {
+        ...resolvedConfig,
+        keystore: {
+          filePath: defaultKeystoreFilePath,
```

Comment:
This addition of a file path to config makes out testing far easier (we can set the keystore file).

Reply:
No, in general, we shouldn't change the product just for testing.

In this case in particular, this can lead to people using this path to point the keystone to a file within their repository, which is something we want to avoid, at least for the first version.